src/components/info/ActivityInfo.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { View } from 'react-native';
+import { Text } from 'react-native-paper';
+
+export const ActivityInfo = () => {
+    return (
+      <View>
+        <Text>ActivityInfo</Text>
+      </View>
+    );
+  }
```

Comment:
All of the new files have CRLF line endings and is missing a newline at the end.